restful_api/myapi/api/schemas/enroll.py

```diff
@@ -11,6 +11,7 @@ class EnrollSchema(ma.SQLAlchemyAutoSchema):
     trainer_id = ma.Int()
     has_passed = ma.Boolean()
     is_official = ma.Boolean()
+    is_approved = ma.Boolean(default=None)
```

Comment:
Why do we use `None` instead of `False` again?

Reply:
We can use `False`, no material difference on logic - but I suppose it's more hygienic for our code. Will make the change.